webmails/roundcube/Dockerfile

```diff
@@ -21,12 +21,14 @@ ENV ROUNDCUBE_URL https://github.com/roundcube/roundcubemail/releases/download/1
 
 ENV CARDDAV_URL https://github.com/mstilkerich/rcmcarddav/releases/download/v4.1.2/carddav-v4.1.2.tar.gz
 
+ENV TZ Etc/UTC
```

Comment:
Please add this line to all Dockerfiles to set a default. 

The 'echo TZ' needs to be moved to entrypoint (jinja template php.ini)

Reply:
When setting the timezone in roundcube.ini (jinja), does the timezone still has to be set in timezone.ini?